Permanent/ViewControllers/Authentication/LoginViewController.swift

```diff
@@ -107,7 +107,18 @@ class LoginViewController: BaseViewController<AuthViewModel> {
             showAlert(title: .success,
                       message: String(format: .emailSent, email!))
         case .error(let message):
-            showAlert(title: .error, message: message)
+            DispatchQueue.main.async {
```

Comment:
Please double check if this async dispatch is really needed. I suppose not

Reply:
Checked and it works great without dispatch.